mallows.py

```diff
@@ -0,0 +1,60 @@
+from marshmallow import Schema
+from marshmallow.fields import Field
+
+
+class MetaMethods:
+    """Separate class for the methods.
+    `Meta` will make new classes inherit from this.
+    """
+
+    def __init__(self, **kwargs):
+        """Custom __init__ method."""
+        # desirialize
+        self.data = self.SCHEMA.load(kwargs)
+        # set attributes
+        for key, val in self.data.items():
+            setattr(self, key, val)
+        for field in self.FIELDS.keys():
+            if not field in self.data.keys():
+                setattr(self, field, None)
+
+    def asdict(self):
+        """Return a dictionary version of the object."""
+        dictionary = self.data
+        for field in self.FIELDS.keys():
+            if not field in self.data.keys():
+                dictionary[field] = None
+        return dictionary
+
+
+class Meta(type):
+    """The meta class.
+    The new class will have these attributes:
+    - `SCHEMA`
+        > This is generated even if the class isn't instantiated.
+        > If you want a nested object, you will always use this
+          like, `Class.SCHEMA`.
+    - `FIELDS`
+        > This is a dictionary of all marshmallow fields inside
+          the class.
+    """
+
+    def __new__(cls, name, bases, attrs):
+        """Custom __new__ method."""
+        new_class = super().__new__(cls, name, bases+(MetaMethods,), attrs)
```

Comment:
This is a hack and messes up the class' MRO because a parent class gets added to the MRO magically, and this has the potential to insert `MetaMethods` more than once if you start mixing your classes together.  If you'll just check Django and DRF, you'll already see there how they do it.